Documentation/ceph-monitoring.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Monitoring
+title: Ceph Monitoring
 weight: 45
```

Comment:
What if we moving the `Ceph Monitoring` and `Ceph Dashboard` topics underneath the `Ceph Storage` in the TOC? A weight of 27 and 28 should do the trick, with `indent: true`.